From 871682649d8cef93724af60011b90e3ca1c33689 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Fri, 25 Sep 2020 02:01:35 -0700
Subject: [PATCH 1669/1851] ASoC: xlnx: i2s: Fix warnings for UNINIT

commit f49aeb00242131cd58d951d160163d250144f492 from
https://github.com/Xilinx/linux-xlnx.git

This patch fix the warnings for the UNINIT type.

The actual warnings for below functions are:

for xlnx_i2s_probe():
Event uninit_use: Using uninitialized value "ch"
Event uninit_use: Using uninitialized value "data_width"

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 sound/soc/xilinx/xlnx_i2s.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/xilinx/xlnx_i2s.c b/sound/soc/xilinx/xlnx_i2s.c
index 1ae53d1cc2ec..31f05a017689 100644
--- a/sound/soc/xilinx/xlnx_i2s.c
+++ b/sound/soc/xilinx/xlnx_i2s.c
@@ -106,7 +106,7 @@ static int xlnx_i2s_probe(struct platform_device *pdev)
 	struct snd_soc_dai_driver *dai_drv;
 	struct xlnx_i2s_dev_data *dev_data;
 	int ret;
-	u32 ch, format, data_width;
+	u32 ch = 0, format, data_width = 0;
 	struct device *dev = &pdev->dev;
 	struct device_node *node = dev->of_node;
 
-- 
2.31.1

